ksql-engine/src/test/java/io/confluent/ksql/KsqlEngineTest.java

```diff
@@ -340,4 +349,22 @@ public void shouldCloseInternallyCreatedTopicClientOnClose() {
     // Then:
     verify(topicClient);
   }
+
+
+  @Test
+  public void shouldParseMultipleStatements() throws IOException {
+    final String statementsString = new String(Files.readAllBytes(
+        Paths.get("src/test/resources/SampleMultilineStatements.sql")), "UTF-8");
+
+    final List<Pair<String, Statement>> parsedStatements =
+        ksqlEngine.parseStatements(statementsString, new MetaStoreImpl(new TestFunctionRegistry()));
+    assertThat(parsedStatements.size(), equalTo(7));
```

Comment:
If you had in the missing indexes you can replace this with:

```
assertThat(parsedStatements, contains(
   instanceOf(CreateStream.class), instanceOf(SetProperty.class), ... etc.
));

Reply:
The order is important too.

Reply:
I do want to check in the specific order!